### ISSUE TITLE
transferring between Nami accounts

### ISSUE THREAD

**Issue by PK3211** [Berry-Pool/nami-wallet#240]
I have all my funds in one account but was wondering how I transfer to another Nami sub account since I think when you delegate it delegates all the funds from that wallet so can't break it up unless split funds between wallets

**Comment by dauta-toast** [comment]
have you tried doing a transaction from one address to the other?

**Comment by PK3211** [comment]
yes it worked, didn't think it would and took some time but did go through
